config/finders/guidance_and_regulation_finder.yml

```diff
@@ -38,6 +38,14 @@ details:
     display_as_result_metadata: false
     filterable: true
     preposition: about
+  - key: topic
+    filter_key: all_part_of_taxonomy_tree
+    name: topic
+    short_name: topic
```

Comment:
Is it ok that this `name` and `short_name` are the same as for the facet above?  What are the `name` and `short_name` actually used for?